sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -692,10 +692,11 @@ ProgramManager::collectDependentDeviceImagesForVirtualFunctions(
   std::set<std::string> HandledSets;
   std::queue<std::string> WorkList;
   for (const sycl_device_binary_property &VFProp : Img.getVirtualFunctions()) {
```

Comment:
Not asking to change it here, but in retrospect I think that the chosen name `getVirtualFunctions` is a bad one. The return value is a property, so the name should convey that.